examples/offscreen/simple_egl.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# vispy: testskip
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+This is a simple osmesa example that produce an image of a cube
+
+If you have both osmesa and normal (X) OpenGL installed, execute with
+something like the following to pickup the OSMesa libraries::
+
+    VISPY_GL_LIB=/opt/osmesa_llvmpipe/lib/libGLESv2.so \
+    LD_LIBRARY_PATH=/opt/osmesa_llvmpipe/lib/ \
+    OSMESA_LIBRARY=/opt/osmesa_llvmpipe/lib/libOSMesa.so \
+    python examples/offscreen/simple_osmesa.py
+"""
+import vispy
+#vispy.use(app='osmesa')  # noqa
+import vispy.app
+
+vispy.app.use_app('egl')
+
+import numpy as np
```

Comment:
This imports brakes flake test, most probably due to not being at the top. Have a look at the osmesa example how to get along without flake8 complaining.

Reply:
See https://travis-ci.org/vispy/vispy/jobs/508119100#L2566-L2571

Reply:
should be done now